### PR TITLE
New version: RustemMussabekov.Raindrop version 5.6.2

### LINKED ISSUE #108256: [Update Request]: RustemMussabekov.Raindrop
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

RustemMussabekov.Raindrop

### Package Version

5.6.2

### Please describe the changes you would like to see

Currently available version in winget -  5.5.23
https://github.com/raindropio/desktop/releases/download/v5.6.2/Raindrop.io-Setup-5.6.2.exe

### PR DESCRIPTION
**Pull request by krystofkrticka**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

* Resolves #108256

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108331)

### CODE REVIEW

File: manifests/r/RustemMussabekov/Raindrop/5.6.2/RustemMussabekov.Raindrop.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+PackageVersion: 5.6.2
+PackageLocale: en-US
+Publisher: Rustem Mussabekov
+PublisherUrl: https://raindrop.io
+PublisherSupportUrl: https://github.com/raindropio/desktop/issues
+PrivacyUrl: https://help.raindrop.io/privacy
+Author: Rustem Mussabekov
+PackageName: Raindrop.io
+PackageUrl: https://raindrop.io/download
+License: MIT
```
Comment (mdanish-kh) [inline manifests/r/RustemMussabekov/Raindrop/5.6.2/RustemMussabekov.Raindrop.locale.en-US.yaml:14]: I can't find a reference to MIT license at the LicenseUrl. If we can't find concrete info about the package license, we should  use `License: Proprietary` instead

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/r/RustemMussabekov/Raindrop/5.6.2/RustemMussabekov.Raindrop.locale.en-US.yaml (hunk 1 above)
Comment (krystofkrticka) [reply manifests/r/RustemMussabekov/Raindrop/5.6.2/RustemMussabekov.Raindrop.locale.en-US.yaml:14]: I've gone though github repos of the app and haven't found anything about license. So I also think that license should be changed to proprietary. I really don't understand where the person who submitted the package for the first time found info that it's licensed under MIT License.

Comment (krystofkrticka) [inline manifests/r/RustemMussabekov/Raindrop/5.6.2/RustemMussabekov.Raindrop.locale.en-US.yaml:14]:
```suggestion
License: Proprietary
```

**Review by mdanish-kh — APPROVED** [review]

File: manifests/r/RustemMussabekov/Raindrop/5.6.2/RustemMussabekov.Raindrop.locale.en-US.yaml (hunk 1 above)
Comment (krystofkrticka) [reply manifests/r/RustemMussabekov/Raindrop/5.6.2/RustemMussabekov.Raindrop.locale.en-US.yaml]: I looked even into history of repo using wayback machine and didn't find any mentions about MIT License.

Comment (mdanish-kh) [reply manifests/r/RustemMussabekov/Raindrop/5.6.2/RustemMussabekov.Raindrop.locale.en-US.yaml]: @krystofkrticka You'll have to mark this conversation as resolved to unblock auto-merge

**Pull request merged**